### PR TITLE
Trigger custom variable expire event on restart

### PR DESCRIPTION
**Pull request by Roger**
### Description of the Change

Trigger the event, when Firebot persist custom variables is enabled, even if Firebot was not running when the expiration happened (if not running, it will trigger the event on the next start).

### Testing

Tested the expiration of the custom variable both with Firebot running and with Firebot closed when the expiration should happen

### CODE REVIEW

File: backend/common/custom-variable-manager.js [hunk 1]
```diff
@@ -7,13 +7,15 @@ const NodeCache = require("node-cache");
 const cache = new NodeCache({ stdTTL: 0, checkperiod: 5 });
 exports._cache = cache;
 
-cache.on("expired", function(key, value) {
+function onCustomVariableExpire(key, value) {
```
Comment (SReject) [inline backend/common/custom-variable-manager.js:10]: use fat arrow functions for methods that are not associated with a `this`:

```js
const onCustomVariableExpire = (key, value) => {
```

**Review by SReject — CHANGES_REQUESTED** [review]

**Comment by SReject** [comment]
Pull requests should fork from v5 and then merge against v5 :

**Comment by ebiggz** [comment]
Thanks for the contribution!

**Comment by Roger** [comment]
Hi! is there any guidelines for contributions? should I then rebase the PR using v5 branch instead of master?

**Comment by ebiggz** [comment]
@Roger We don't have contributor guidelines at this time, unfortunately. Sorry about that! We discussed getting some defined here soon. 

But yes, rebasing to v5 would be great. We currently have it such that `master` represents the latest stable release and `v5` is our active development branch. When we are ready for a release, we merge to `master` and our CICD makes a build for us.

**Comment by Roger** [comment]
Sorry I pushed an stale v5 branch from my fork, I just re-pushed updating from upstream and thanks for changing the base branch to v5!

**Review by ebiggz — APPROVED** [review]

**Comment by ebiggz** [comment]
Looks great. I'll get this merged so it can be in the next stable release. Thanks again for contributing!

**Pull request merged**